_data/staff.yml

```diff
@@ -162,13 +162,10 @@ moderators:
       id: 3a3f756e-7925-409a-be5e-e8c1694b3e2c
     - username: Cazadorian
       id: 6863869b-4b8c-4445-b778-a8e016775ae4
-junior moderators:
-  color: "#f5f"
-  members:
-    - username: Chiddy_
-      id: 2fde9fe3-8aa0-43bf-9f74-e1c46f4dae72
     - username: catzzy
       id: 1f7851a8-3a58-4d68-839f-939727a11080
+    - username: Chiddy_
+      id: 2fde9fe3-8aa0-43bf-9f74-e1c46f4dae72
```

Comment:
Will this work?
```suggestion
    - username: Chiddy_
      id: 2fde9fe3-8aa0-43bf-9f74-e1c46f4dae72
```
```suggestion
    - username: Chiddy_
      id: 2fde9fe3-8aa0-43bf-9f74-e1c46f4dae72
junior moderators:
  color: "#f5f"
  members: []
```

Reply:
That should work fine

![Image 2022-02-02 at 6 11 25 PM](https://user-images.githubusercontent.com/6914518/152264582-15e71623-c463-4a30-b3a1-b3540312fef9.jpg)